lib/metadataTypes/definitions/Interaction.definition.js

```diff
@@ -128,9 +128,9 @@ module.exports = {
         },
         modifiedDate: {
             isCreateable: false,
-            isUpdateable: false,
+            isUpdateable: true,
```

Comment:
explain?

Reply:
because the docs say you need to pass it in for updates:
"Although the modifiedDate may not be updated, you are required to pass current value that matches the database record."
>> https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/putUpdateInteraction.html